### PR TITLE
Feat/hot keys

### PR DESCRIPTION
**Pull request by EdPrys**
Fix: #8 
add: hot-keys

### CODE REVIEW

File: common/utils/hotKeys.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+        case 'down':
+          videoEl.volume = Math.max(videoEl.volume - 0.05, 0);
+          event.preventDefault();
+          break;
+        case 'left':
+          videoEl.currentTime = Math.max(videoEl.currentTime - 5, 0);
+          event.preventDefault();
+          break;
+        case 'right':
+          videoEl.currentTime = videoEl.currentTime + 5;
```
Comment (Security2431) [inline common/utils/hotKeys.ts:53]: it scrolls on 14s 🤔 Can you figure out?

File: common/utils/hotKeys.ts [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+        case 'up':
+          videoEl.volume = Math.min(videoEl.volume + 0.05, 1);
+          event.preventDefault();
+          break;
+        case 'down':
+          videoEl.volume = Math.max(videoEl.volume - 0.05, 0);
+          event.preventDefault();
+          break;
+        case 'left':
+          videoEl.currentTime = Math.max(videoEl.currentTime - 5, 0);
```
Comment (Security2431) [inline common/utils/hotKeys.ts:49]: it scrolls on 14s 🤔 Can you figure out?

Comment (Security2431) [reply common/utils/hotKeys.ts:49]: Try to scroll at the end video

File: common/utils/hotKeys.ts [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        case '8':
+          videoEl.currentTime = videoEl.duration * 0.8;
+          event.preventDefault();
+          break;
+        case '9':
+          videoEl.currentTime = videoEl.duration * 0.9;
+          event.preventDefault();
+          break;
+        case 'shift+n':
+          // navigateNext()
```
Comment (Security2431) [inline common/utils/hotKeys.ts:97]: please add comment what this hot key should do

File: common/utils/hotKeys.ts [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        case 'c':
+          if (videoEl instanceof HTMLVideoElement) {
+            const firstTextTrack = videoEl.textTracks[0];
+            if (firstTextTrack) {
+              firstTextTrack.mode =
+                firstTextTrack.mode === 'showing' ? 'disabled' : 'showing';
+            }
+          }
+          event.preventDefault();
+          break;
```
Comment (Security2431) [inline common/utils/hotKeys.ts:33]: What this hotkey should do?

File: common/utils/hotKeys.ts (hunk 2 above)
Comment (Security2431) [reply common/utils/hotKeys.ts:49]: At the beginning video it scrolls by 5s, but then by 14s

**Review by Security2431 — CHANGES_REQUESTED** [review]
Looks good, left some comments. Also, please, can you base this PR on main branch? Otherwise, we cannot merge it, before other PRs merged

File: common/utils/hotKeys.ts (hunk 4 above)
Comment (EdPrys) [reply common/utils/hotKeys.ts:33]: turn subtitles on and off

File: common/utils/hotKeys.ts [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+        case 'j':
+          videoEl.currentTime = Math.max(videoEl.currentTime - 15, 0);
+          event.preventDefault();
+          break;
+        case 'l':
+          videoEl.currentTime = videoEl.currentTime + 15;
+          event.preventDefault();
+          break;
+        case 'c':
+          if (videoEl instanceof HTMLVideoElement) {
```
Comment (Security2431) [inline common/utils/hotKeys.ts:25]:
```suggestion
          // Show/hide subtitles
          if (videoEl instanceof HTMLVideoElement) {
```

File: common/utils/hotKeys.ts (hunk 1 above)
Comment (Security2431) [reply common/utils/hotKeys.ts:55]: Created separate issue #28

File: common/utils/hotKeys.ts (hunk 2 above)
Comment (Security2431) [reply common/utils/hotKeys.ts]: Moved this to separate issue #28

File: components/VideoPlayer/VideoPlayer.tsx [hunk 6]
```diff
@@ -29,6 +30,7 @@ const VideoPlayer: React.FC<Props> = ({ videoUrl }) => {
         });
 
         await player.load(videoUrl);
+        setupHotKeys(videoRef.current!);
```
[the remark this answers is not in the document]
Comment (Security2431) [reply components/VideoPlayer/VideoPlayer.tsx:33]: If you want clarify, feel free to ping me

**Pull request merged**